doc/releases/migration-guide-3.6.rst

```diff
@@ -76,6 +76,10 @@ Bluetooth
   cleared on :c:func:`bt_enable`. Callbacks can now be registered before the initial
   call to :c:func:`bt_enable`, and should no longer be re-registered after a :c:func:`bt_disable`
   :c:func:`bt_enable` cycle.
+* The Bluetooth Mesh ``model`` declaration has been changed to add prefix ``const``.
+  The ``model->user_data``, ``model->elem_idx`` and ``model->mod_idx`` field has been changed to
+  the new runtime structure, replayced by ``model->rt->user_data``, ``model->rt->elem_idx`` and
```

Comment:
```suggestion
  the new runtime structure, replaced by ``model->rt->user_data``, ``model->rt->elem_idx`` and
```
+ please move under `Mesh` subsection (e.g.: https://github.com/zephyrproject-rtos/zephyr/blob/063ce9caf54fa656f02ae48f3c9d537659a10dec/doc/releases/release-notes-3.5.rst?plain=1#L201)

Reply:
you mean move this to release note file instead of migration file?

Reply:
@PavelVPV @alxelax Done. Take a look again, thanks.